src/main/java/com/github/tomakehurst/wiremock/core/WireMockConfiguration.java

```diff
@@ -526,7 +545,7 @@ public ProxySettings proxyVia() {
   @Override
   public Stores getStores() {
     if (stores == null) {
-      stores = new DefaultStores(filesRoot);
+      stores = new DefaultStores(filesRoot, proxyPassThrough);
```

Comment:
I feel like `proxyPassThrough` shouldn't be passed in here.

We want the `proxyPassThrough` default value to be used regardless of whether we're using the default stores implementation or one supplied by the user.

Reply:
incorporated